README.md

```diff
@@ -49,24 +49,21 @@ Installation of Nexus IQ Server application is to `/opt/sonatype/nexus-iq-server
 
 By default, the IQ Server reads its [main configuration file](https://help.sonatype.com/iqserver/configuring/config.yml) from within the image at `/etc/nexus-iq-server/config.yml`.
 
-There is an environment variable `JAVA_OPTS` that is being used to pass JVM arguments to the java command that launches IQ server. The default value is `-Djava.util.prefs.userRoot=${SONATYPE_WORK}/javaprefs`.
+There is an environment variable `JAVA_OPTS` that passes JVM arguments to the java command that launches IQ server.
 
 This environment variable can be adjusted at runtime:
 
 ```
-$ docker run -d -p 8070:8070 -p 8071:8071 --name nexus-iq-server -e JAVA_OPTS="-Djava.util.prefs.userRoot=/some-other-dir" sonatype/nexus-iq-server
+$ docker run -d -p 8070:8070 -p 8071:8071 --name nexus-iq-server -e JAVA_OPTS="-Doption=value" sonatype/nexus-iq-server
 ```
 
-Of particular note, `-Djava.util.prefs.userRoot=/some-other-dir` can be set to a persistent path, which will maintain
-the installed Nexus IQ Server License if the container is restarted.
-
 Further, you can [customize parts of the default config.yml settings by using Java system properties](https://help.sonatype.com/iqserver/configuring/advanced-server-configuration) defined inside the environment variable.
 
 Example: To customize the HTTP proxy server that IQ Server will use to make outbound requests:
 
 ```
 $ docker run -d -p 8070:8070 -p 8071:8071 --name nexus-iq-server -e \
-  JAVA_OPTS="-Ddw.proxy.hostname=proxy.example.com -Ddw.proxy.port=8888 -Djava.util.prefs.userRoot=/some-other-dir" sonatype/nexus-iq-server
+  JAVA_OPTS="-Ddw.proxy.hostname=proxy.example.com -Ddw.proxy.port=8888 sonatype/nexus-iq-server
```

Comment:
We seem to have lost the closing double-quotes here.

Reply:
> We seem to have lost the closing double-quotes here.

oops added back in https://github.com/sonatype/docker-nexus-iq-server/pull/46/commits/62b73d3a5891d70bf5f91accb04921038597e481 https://github.com/sonatype/docker-nexus-iq-server/pull/46/commits/7d74dfb9ff6a709dd4032f73af6578abd20ece8c